Social media (#11)
jhudsl/cheatsheets: social_media.Rmd

---
title: |
  <center> <u>Social Media </u></center>
  <center> <p style = "color:#14395f;"> Tips and Tricks</p></center>
date: "Last updated: `r format(Sys.time(), '%B %d, %Y')`"
output:
  html_document:
    number_sections: false
css: ITN_style.css
---

<hr>

<div class="row">

<div class="column">

# Best Practices:

<input type="checkbox">  Check that you have been transparent 

>   - “I’m a scientist @FredHutch. Opinions are my own, retweets aren't endorsements.”

Reviewer: ```suggestion
  - “I’m a scientist @FredHutch. Opinions are my own, sharing other's posts isn't an endorsement.”
```

Reviewer: Trying to make it less platform specific

Author: how about
"Opinions are my own, reposts aren't endorsements."

Reviewer: Love it!

Author: thanks for the suggestion!

> <input type="checkbox">  Check that you have cited and/or tagged sources 

Reviewer: ```suggestion
<input type="checkbox">  Check that you have cited and/or tagged sources and collaborators
```

Reviewer: Should you ask for permission before tagging a collaborator? Honestly don't know

<input type="checkbox">  Check that you are not violating patient or intellectual privacy

<input type="checkbox">  Check if you are using sarcasm or tone that might be lost in translation

<input type="checkbox">  Check that you have added a creative element

  - Use stories, anecdotes, images, videos, art, GIFs, poetry, [cartoons](https://getavataaars.com/), [memes](https://giphy.com/categories/memes)
  - Be cautious about memes - may not target right audience

<input type="checkbox">  If you are unsure, don't post it!

</div>

<div class="column">
# Content to Share:
 
- Walk through how to do certain tasks with your software
- Report releases, new features, papers etc.

> - Describe your work & your colleagues work

Reviewer: ```suggestion
- Describe your work & your colleagues' work
```

<hr>

# Tools:
- [Buffer](https://buffer.com/)
  - Schedule your content to the most popular platforms including Facebook, Instagram, TikTok, LinkedIn, Threads, Bluesky, YouTube Shorts, Pinterest, Google Business, Mastodon and X.
  - Collect engagement metrics and sorts by negative and/positive feedback
- [RSS feeds](https://rss.com/blog/how-do-rss-feeds-work/) and [Tiny Tiny RSS](https://tt-rss.org/)
  - Host feeds for free and aggregate and distribute them
- [Substack](https://substack.com/about)
  - Create blogs or newsletters that updates subscribers, has embedded tracking of engagement (works with RSS feeds)
- [Newsletters on LinkedIN](https://www.linkedin.com/help/linkedin/answer/a517925)
  - Separate from the organization profile, allows for user subscriptions and engage with comments and questions

<hr>

# Tips:

- **Efficiency**: 
  - Archive everything so other team members can access and review to help with inspiration and reposting
  - Repurpose posts for different audiences
  - Plan and schedule in advance
  - Use [Buffer](https://buffer.com/) or other tools to post the same content on multiple platforms
  - Templates can be really useful - quickly looks professional

>   - LLMs can help write content

Reviewer: "draft content" and note that you want to review it?

Author: added a new check about it
